kiosk-helm/Chart.yaml

```diff
@@ -0,0 +1,27 @@
+# SPDX-License-Identifier: Apache-2.0
+#!BuildTag: suse/kiosk/kiosk-helm:latest
```

Comment:
we most often use "-chart" as suffix instead of -helm. can we change it here (yes I know `rmt-helm` is the exception to the rule..)